### PR TITLE
create B827EBFFFEEA03F1.json

### PR DESCRIPTION
**Pull request by DuyTran2626**
{"gateway_conf": { 
		"gateway_ID": "B827EBFFFEEA03F1", 
		"servers": [ 
			{ 
				"server_address": "thethings.meshed.com.au", 
				"serv_port_up": 1700, 
				"serv_port_down": 1700, 
				"serv_enabled": true 
			} 
		], 
		"ref_latitude": 10.78616388, 
		"ref_longitude": 106.66710780, 
		"ref_altitude": 40, 
		"contact_email": "trandinhduy96er@gmail.com", 
		"description": "Gateway Lora Demo" 
	} 
}

### CODE REVIEW

File: B827EBFFFEEA03F1.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (gonzalocasas) [inline B827EBFFFEEA03F1.json]: This is missing an opening curly brace.

**Comment by DuyTran2626** [comment]
i added opening curly brace.

**Comment by gonzalocasas** [comment]
@DuyTran2626 could you please add it to the file content, not the comment? ;) thx

**Comment by DuyTran2626** [comment]
can you delete that pull request for me? i can't find where to add it. Many
thanks

**Comment by DuyTran2626** [comment]
i added it into my file :)

**Comment by gonzalocasas** [comment]
Thanks! merged!

**Comment by DuyTran2626** [comment]
thank you so much. My gateway is working :)))

**Pull request merged**